assets/schema/ensemble_schema.json

```diff
@@ -823,6 +823,14 @@
           "type": "boolean",
           "description": "a shining animation to designate that the content is loading. The animation can be over a defaults shimmer or the loadingWidget if specified."
         },
+        "baseColor": {
```

Comment:
these 2 styles related shoudl be inside "styles". This is sort of how other widgets works.